knimin/lib/data_access.py

```diff
@@ -864,6 +864,17 @@ def format_survey_data(self, md, external_surveys=None, full=False):  # noqa
                     bc_info['ag_login_id'] + participant_name).hexdigest()
                 md[1][barcode]['PUBLIC'] = 'Yes'
 
+                # Convert finer grained IBD to coarser grained
+                ibd = md[1][barcode].get('IBD_DIAGNOSIS_REFINED',
+                                         'Unspecified')
+                if ibd != 'Unspecified':
+                    if ibd in {"Ileal Crohn's Disease",
+                               "Colonic Crohn's Disease",
+                               "Ileal and Colonic Crohn's Disease"}:
+                        md[1][barcode]['IBD_DIAGNOSIS'] = "Crohn's disease"
+                    elif ibd == 'Ulcerative colitis':
+                        md[1][barcode]['IBD_DIAGNOSIS'] = 'Ulcerative colitis'
+
```

Comment:
Why have it as a category if it is undefined?  How will it be dented in pull-down?